distributed/rpc/pipeline/README.md

```diff
@@ -0,0 +1,9 @@
+Distributed Pipeline Parallel Example
+
+This example shows how to distribute a ResNet50 model on two RPC workers and
+then implement distributed pipeline parallelism using RPC.
```

Comment:
Should we include a quick description of the pipelining strategy (pipelining micro-batches within a batch and then synchronously running the optimizer step)? Since this is like GPipe, should we also link the paper here?